CMakeLists.txt

```diff
@@ -510,7 +511,6 @@ elseif( UNIX AND NOT APPLE )
   set(OS_COMPILE_DEFINITIONS
     ${ARCH_COMPILE_DEFINITIONS}
     LINUX=1
-    RELEASE=1
```

Comment:
Curious: Doesn't this need to be replaced with something like

```
 $<IF:$<CONFIG:DEBUG>,DEVELOPMENT,RELEASE>=1
```

as below?

Reply:
The `$<IF>` below applies on all platforms, so it doesn't need to be repeated in the `OS_COMPILE_DEFINITIONS` for Linux.

On a related note though, on Windows I had somehow added a global compile switch. Moved into `OS_COMPILE_DEFINITIONS` now for consistency.